.internal-ci/docker/Dockerfile.fogview

```diff
@@ -12,6 +12,7 @@ FROM ${REPO_ORG}/runtime-base:${BASE_TAG}
 ARG RUST_BIN_PATH=target/release
 COPY ${RUST_BIN_PATH}/libview-enclave.signed.so /usr/bin/
 COPY ${RUST_BIN_PATH}/fog_view_server /usr/bin/
```

Comment:
In fog_ledger `fog_ledger_router` replaces the `fog_ledger_server`  should this be the same here? Where fog_view_server should be removed.

Reply:
Without taking a position on what the names should be, these are the names that are currently being used.

`fog_view_server` - the Fog View store server
`fog_view_router` - the Fog View router server
`key_image_store` - the Fog Ledger Key Image store server
`fog_ledger_router` - the Fog Ledger router server